.github/workflows/prettier-check.yml

```diff
@@ -0,0 +1,29 @@
+name: "Prettier Check"
+
+on:
+  pull_request:
+
+jobs:
+  check-prettier:
+    name: Prettier Check
+    runs-on: ubuntu-latest
+    steps:
+      - name: Download repo
+        uses: actions/checkout@v4
+
+      - name: Install pnpm
+        uses: pnpm/action-setup@v4
+        with:
+          version: 9
+
+      - name: Use Node.js ${{ matrix.node-version }}
```

Comment:
am I missing something? where's the matrix version?

Reply:
Oops, yeah, thanks for noticing that.